addons/func_godot/src/netradiant_custom/netradiant_custom_gamepack_config.gd

```diff
@@ -208,10 +214,15 @@ func do_export_file() -> void:
 	print("Exporting NetRadiant Custom Default Buld Menu to ", target_file_path)
 	file = FileAccess.open(target_file_path, FileAccess.WRITE)
 	if file != null:
-		file.store_string("<?xml version=\"1.0\"?><project version=\"2.0\"></project>")
-		file.close()
-	else:
-		printerr("Error: Could not modify " + target_file_path)
+		file.store_string("<?xml version=\"1.0\"?>\n<project version=\"2.0\">\n")
+		
+		for key in build_xml_variables.keys():
+			file.store_string('\t<var name="%s">%s</var>\n' % [key, build_xml_variables[key]])
```

Comment:
Should do a safety check to make sure we're using a string here.

```gdscript
		for key in default_build_menu_variables.keys():
			if default_build_menu_variables[key] is String:
				file.store_string('\t<var name="%s">%s</var>\n' % [key, build_xml_variables[key]])
			else:
				push_error("ERROR: Build menu variable \"" + key + "\" is not a String")

Reply:
Sure - %s casts to string I thought but it should still probably be caught anyway.

Reply:
AFAIK not all types can be cast to String though. I know there's a special `str(value)` in GDScript but does `%s` use that to convert it to String or does it just do the equivalent of `value as String`?

Reply:
I'm not sure, actually, so that's a great point. Will fix